rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json

```diff
@@ -1169,8 +1169,33 @@
               "bf:Production"
             ]
           },
+          "place": {
```

Comment:
can you add a minItems properties at each time you have an array?

Reply:
ok